rules/github-scrum-workflow/rule.md

```diff
@@ -0,0 +1,79 @@
+---
+type: rule
+archivedreason:
+title: How to set up GitHub scrum workflow
+guid: e63da194-c5d2-4830-9106-d164aafcddb5
+uri: do-you-know-the-how-to-set-up-github-workflow
+created: {date_created}
+authors: []
+related:
+
+---
+### How to set up GitHub scrum workflow
+
+There are several ways of turning your GitHub repository into a simple but powerful agile work environment, the following is the recommended:
+
+<!--endintro-->
+
+#### How it works
+
+- Product Backlog Items (PBIs) are reported as **issues**
+- Points and meta data are assigned to items as **labels**
+- **milestones** are used to group issues in Sprints
+
+#### 1. Create issues as backlog items
+
+To create a new backlog item, just create a new issue.
+
+Once a new issue has been created, assign it the right labels and/or assign it to a sprint (milestone).
+
+Issues allow you to have a conversation about the item and even allow you to create task lists inside the issue using [GitHub's markdown](https://guides.github.com/features/mastering-markdown/).
+
+#### 2. Add labels to issues
+
+Add the following labels to your repository:
+
+### Estimates
+
+`estimate` labels allow you to set estimates in your backlog e.g.:
```

Comment:
estimates should align with https://www.ssw.com.au/rules/estimating-do-you-know-how-to-size-user-stories-effectively

Reply:
> estimates should align with https://www.ssw.com.au/rules/estimating-do-you-know-how-to-size-user-stories-effectively

Done - referenced rules to align estimates